src/app/components/update-itinerary/update-itinerary.component.html

```diff
@@ -0,0 +1,70 @@
+<nz-modal [nzVisible]="true" nzTitle="Update Itinerary"  (nzOnOk)="onSubmit()" (nzOnCancel)="handleCancel()">
+  <ng-container *nzModalContent>
```

Comment:
This looks like the other component. Why repeat yourself? Create a component that handles these for you and reuse that.